Create ui-c8y-1021-22-98-data-point-table-widget-display-issue-max-min-values-9425-graft.md (#3480)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-22-98-data-point-table-widget-display-issue-max-min-values-9425-graft.md

---
date: ""

> title: Fixed data point table widget not displaying maximum and minimum values correctly

Reviewer: ```suggestion
title: Data point table widget now correctly displays maximum and minimum values
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63958
version: 1021.22.98
---

> When displaying the minimum and maximum values for a data point at the same time was selected, the maximum values were showing the minimum values instead. This setting now shows the maximum value correctly.

Reviewer: ```suggestion
In the "Data point" widget, when displaying both the minimum and maximum values for a data point was selected, the maximum values showed the minimum values instead. This issue has been fixed. When selecting this setting, the maximum values are now shown correctly.
```

> In the "Data point" widget, when displaying both the minimum and maximum values for a data point was selected, the maximum values showed the minimum values instead. This issue has been fixed. When selecting this setting, the maximum values are now shown correctly.

Reviewer: ```suggestion
In the "Data point table" widget, when displaying both the minimum and maximum values for a data point was selected, the maximum values showed the minimum values instead. This issue has been fixed. When selecting this setting, the maximum values are now shown correctly.
```

---
date: ""
title: Data Point Table Widget Display Issue (Max/Min Values) (#9425) [GRAFT][release/y2025] (#9433)
product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-63958
version: 1021.22.98
---
Co-authored-by: Lukasz Janusz <114157358+luja-c8y@users.noreply.github.com>

> Co-authored-by: Lukasz Janusz <lukasz.janusz@cumuloicty.com>

Reviewer 2: ```suggestion
When displaying the minimum and maximum values for a data point at the same time was selected, the maximum values were showing the minimum values instead. This setting now shows the maximum value correctly.
```
